metabase/metabase
# Remove brand color from CSS build process

Related to https://github.com/metabase/metabase-private/issues/73

Removes static `--color-brand` and passes it via `GlobalStyles`. 

How to verify:
- The app should look normal, with the following places special-cased:

1. Button - make sure the background color of primary buttons changes 
<img width="576" alt="Screenshot 2023-08-04 at 15 41 04" src="https://github.com/metabase/metabase/assets/8542534/b25fbfd5-98ea-4876-99a1-fb6d5ab3c6c7">

2. TableInteractive - table viz, hover effects
<img width="1221" alt="Screenshot 2023-08-04 at 15 41 23" src="https://github.com/metabase/metabase/assets/8542534/d02a9bf2-39cb-4ec1-a713-9ca073a8fe5a">

3. ContentTable - group list in Admin -> People, row hover effect
<img width="1728" alt="Screenshot 2023-08-04 at 15 40 58" src="https://github.com/metabase/metabase/assets/8542534/00ed24f0-191b-4ba0-9236-069755177d49">
